test/Project.toml

```diff
@@ -13,6 +13,7 @@ Memento = "f28f55f0-a522-5efc-85c2-fe41dfb9b2d9"
 OSQP = "ab2f91bb-94b4-55e3-9ba0-7f65df51de79"
 ParameterJuMP = "774612a8-9878-5177-865a-ca53ae2495f9"
 PowerModels = "c36e90e8-916a-50a6-bd94-075b64ef4655"
+PowerSimulations = "e690365d-45e2-57bb-ac84-44ba829e73c4"
```

Comment:
This file shouldn't be checked in

Reply:
Why not? the test environment needs it. @daniel-thom also mentioned that it shouldn't need to be in the Project, but without it, the user needs to `dev .` in the test environment to do anything. This resolves that silliness... does it create a problem?

Reply:
This line should be removed.

The test environment does not need this if you run Pkg.test().  The PowerSimulations package is added implicitly.  This is only a problem if you `include("test/runtests.jl")`.  If the user wants to use that non-Julia-standardized workflow (which I happen to like) then they need to perform `]dev .`.  Doing it this way means that the test will download PowerSimulations.jl instead of using the current directory.